server/go/services/logcollector/cmd/docker/Dockerfile

```
@@ -23,7 +23,7 @@ RUN mkdir /app
 WORKDIR /app
 
 # Build stages requires defining GO_VERSION again
-ARG GO_VERSION=1.23
+ARG GO_VERSION=1.25
```

No need for defining the default again, just to redeclare the arg:
```suggestion
ARG GO_VERSION
```